### PR TITLE
Change analystic ID, update cytosu and add soyokaze!

### PR DESCRIPTION
**Pull request by HelloYeew**
# TODO

- [x] Update cytosu new mod
- [x] Add soyokaze
- [x] Add osu!DIVA **(push to next PR)**
- [x] Add touhosu **(push to next PR)**
- [x] Add sublink
- [x] Update status page
- [x] Update mvis to new version (This already deploy in build source code but not deploy in main source code now.)

# TODO (for better update time)

_I will move this to new PR._

## Prologue

Since we add a night mode that can invert a picture when that picture is black, it affect a lot to a netlify and framework build since we must add a CSS class that make that picture can invert color on night mode _by hand_. So I think since we add night mode we can only build a website by hand only. That's why I decrease a number of my pull request down since then and currently I update a status page directly to the build repo (And it affect other page to bug on a date.)

It's ok if wiki update like 5-10 minutes late. But we have a page that need a real-time update, **a status page**. It's not worth like "Oh, sentakki have a new build! We must build a website again and put a CSS class to a picture that require color inversion" So I think I will migrate a status page to a new repositories since it need a lot of commit when a new build is up.  And when we want to update that page it, just commit to that repositories, I will open auto-build on that commit too. A migration will happen slowly to not make a link in website like broke and have `404 not found`. Here is a migration plan.

### Make a new place for status page
- [x] Transfer auto build and deploy on this repositories to a dev link (https://dev.rulesets.info)
- [ ] Build a new repositories only for status pages.

### Migration
- [ ] Build a website (after I finish make a 3 new rulesets page above and push it to a main website source code)
- [ ] Pull a code that already build on status page to a new repositories.
- [ ] Bind an url that we already have on status page (https://status.rulesets.info) to a new page.
- [ ] Change a code on header to a new link in both a source code repo and build repo.
- [ ] Check that it's working.

### Destroy status update route from both repositories
- [ ] Delete status page folder from source code repo.

### CODE REVIEW

File: content/posts/cytosu/index.md [hunk 1]
```diff
@@ -41,8 +41,28 @@ Cytosu based gameplay design for touch screen but can be played on tablet or mou
...
+    - Increases speed 1.50x.
+  - Score Multiplier: 1.12x
+  - Click the icon again for `Nightcore Mod` : Uguuuuuuuu...
+    - Increases speed 1.50x and raises audio pitch.
+
 ### automation
 
+- ![Relax Icon](mod-icon/relax-mod.png) Relax
+  - To lazy to click your keyboard, eh?
+    - Just move your mouse to a circle note.
```
Comment (IKeepItNoodles) [inline content/posts/cytosu/index.md:64]:
```suggestion
- ![Relax Icon](mod-icon/relax-mod.png) Relax
  - Too lazy to click your keyboard, eh?
    - You only need to move your cursor to the notes, you do not have to click them.
```

File: content/posts/soyokaze/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+- Creator : [Alden Wu](https://github.com/goodtrailer)
+- Availability : Available publicly
+- GitHub Repositories : [goodtrailer/soyokaze](https://github.com/goodtrailer/soyokaze)
+
+Soyokaze! is a custom ruleset mimicking Genshin Impact's Ballads of Breeze mini-game. The ruleset is developed by Alden Wu.
+
+# gameplay
+
+Primarily in soyokaze! have 2 rhythm rings. A rhythm ring is like DPAD, it have 4 side (north, ease, south and west). A note will gradually be appear at the four side of the ring and then disappear. Player must press a note in time before it disappear.
```
Comment (IKeepItNoodles) [inline content/posts/soyokaze/index.md:24]:
```suggestion
Primarily in soyokaze! the player has 2 rhythm rings. A rhythm ring is like DPAD, it have 4 side (north, east, south and west). A note will appear at one of the four sides of both rings and then disappear. The player must press the note in time before it disappears using the keyboard button assigned for this specific note placement.
```

File: content/posts/soyokaze/index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- Button 6 (DPAD-R ←) - `J`
+- Button 7 (DPAD-R ↓) - `K`
+- Button 8 (DPAD-R →) - `L`
+
+# mods
+
+### difficulty reduction
+
+- ![Easy Icon](mod-icon/easy-mod.png) Easy
+  - Larger circles, more forgiving HP drain, less accuracy required, and multiple lives. As a bonus, it make the map impossible to read!
```
Comment (IKeepItNoodles) [inline content/posts/soyokaze/index.md:46]:
```suggestion
  - Larger circles, more forgiving HP drain, less accuracy required, and multiple lives. As a bonus, it makes the map impossible to read!
```

File: content/posts/soyokaze/index.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+  - Score Multiplier: 0.85x
+
+- ![No Fail Icon](mod-icon/no-fail-mod.png) No Fail
+  - You can't fail, no matter what.
+    - Make it impossoble to fail.
+  - Score Multiplier: 0.50x
```
Comment (IKeepItNoodles) [inline content/posts/soyokaze/index.md:52]:
```suggestion
- ![No Fail Icon](mod-icon/no-fail-mod.png) No Fail
  - You can't fail, no matter what.
    - Makes it impossible to fail.
  - Score Multiplier: 0.50x
```

File: content/posts/soyokaze/index.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  - Watch a perfect automated play through the song.
+
+### conversion
+
+- ![Random Icon](mod-icon/random-mod.png) Random
+  - Shuffle around the notes!
+
+- ![Difficulty Adjust Icon](mod-icon/difficulty-adjust-mod.png) Difficulty Adjust
+  - Override a beatmap's difficulty settings.
+  - You can override *HP Drain*, *Accuracy*, *Paddle Size*, *Approach Rate* and *Extended Limits*
```
Comment (IKeepItNoodles) [inline content/posts/soyokaze/index.md:91]:
```suggestion
- ![Difficulty Adjust Icon](mod-icon/difficulty-adjust-mod.png) Difficulty Adjust
  - Override a beatmap's difficulty settings.
  - You can override *Circle Size*, *HP Drain*, *Accuracy*, *Approach Rate* and *Extended Limits*
```

File: content/posts/soyokaze/index.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+- `Hit Circle gap` by slider
+- `Judgement distance from screen center` by slider
+- `Judgement gap` by slider
+- `Input Overlay distance from screen center` by slider
+- `Input Overlay gap` by slider
+- `Kiai Visualizer distance from screen center` by slider
+
+# demo
+
+{{< youtube id="hWjG0W7EiAE" >}}
```
Comment (IKeepItNoodles) [inline content/posts/soyokaze/index.md:118]:
```suggestion
# demo

{{< youtube id="hWjG0W7EiAE" >}}
```
```suggestion
# preview

{{< youtube id="hWjG0W7EiAE" >}}
```

File: content/posts/soyokaze/index.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+- `Show Input Overlay` by toggle button
+- `Show Kiai Visualizer` by toggle button
+- `Hit Circle distance from screen center` by slider
+- `Hit Circle gap` by slider
+- `Judgement distance from screen center` by slider
+- `Judgement gap` by slider
+- `Input Overlay distance from screen center` by slider
+- `Input Overlay gap` by slider
+- `Kiai Visualizer distance from screen center` by slider
+
```
Comment (IKeepItNoodles) [inline content/posts/soyokaze/index.md:115]:
```suggestion
- `Show Input Overlay` by a toggle button
- `Show Kiai Visualizer` by a toggle button
- `Hit Circle distance from screen center` by a slider
- `Hit Circle gap` by a slider
- `Judgement distance from screen center` by a slider
- `Judgement gap` by a slider
- `Input Overlay distance from screen center` by a slider
- `Input Overlay gap` by a slider
- `Kiai Visualizer distance from screen center` by a slider

```

File: content/posts/cytosu/index.md [hunk 8]
```diff
@@ -41,8 +41,28 @@ Cytosu based gameplay design for touch screen but can be played on tablet or mou
     - Makes it impossible to fail.
   - Score Multiplier: 0.50x
 
+- ![Half Time Icon](mod-icon/half-time-mod.png) Half Time
+  - Less Zoom...
+    - Decreases speed .75x.
+  - Score Multiplier: 0.30x
+  - Click the icon again for*Daycore Mod* : Whoaaaaa...
```
Comment (IKeepItNoodles) [inline content/posts/cytosu/index.md:48]:
```suggestion
  - Click the icon again for `Daycore Mod` : Whoaaaaa...
```

I think it should be like this no?

File: content/posts/soyokaze/index.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
+---
+title: "soyokaze!"
+img: "soyokaze-logo.png"
+date: 2021-04-20
+summary: "An osu! ruleset mimicking Genshin Impact&rsquo;s Ballads of Breeze mini-game."
```
Comment (IKeepItNoodles) [inline content/posts/soyokaze/index.md:5]:
```suggestion
summary: "An osu! ruleset mimicking Genshin Impact's Ballads of Breeze mini-game."
```
I am not sure what happened here but I think this is wrong

Comment (HelloYeew) [reply content/posts/soyokaze/index.md:5]: It's HTML numberic code for curly quote.

File: content/pages/status/index.md [hunk 10]
```diff
@@ -24,12 +24,13 @@ Last Update : May 6, 2021 17:28 UTC
 | [mvis](/posts/mvis)  | ✅  | Everything works | 2021.504.1 (May 4, 2021) | [⬇️](https://github.com/EVAST9919/lazer-m-vis/releases/latest) |
 | [hitokori](/posts/hitokori)  | ✅ | Everything works | 0.18.5 (April 26, 2021) | [⬇️](https://github.com/Flutterish/Hitokori/releases/latest) |
 | [bosu](/posts/bosu)  | ✅ | Everything works | 2021.502.0 (May 2, 2021) | [⬇️](https://github.com/EVAST9919/bosu/releases/latest) |
-| [cytosu](/posts/cytosu)  | ⚠️ | Autoplay mod causes client to crash. | 2021.417.0 (April 17, 2021)| [⬇️](https://github.com/GSculerlor/Cytosu/releases/latest) |
+| [cytosu](/posts/cytosu)  | ✅ | Everything works | 2021.417.0 (April 17, 2021)| [⬇️](https://github.com/GSculerlor/Cytosu/releases/latest) |
```
Comment (LumpBloom7) [inline content/pages/status/index.md:27]: The status changed but version didn't. Was it a mistake previously?

Comment (HelloYeew) [reply content/pages/status/index.md:27]: I test a ruleset on last version of osu!lazer and it's like in status that I write, then I try to install it again and it's work perfectly. It's my fault not check it correctly about rulesets version.

**Pull request merged**